documentation/manual/src/ornate/tutorial-openapi.md

```diff
@@ -158,13 +156,9 @@ Finally, we start a `NettyServer` and give it both the `CounterServer` and
 ## Summary
 
 To generate an OpenAPI definition from your service description this one
-must be written using a different set of algebra interfaces, which live
-in the `endpoints.documented.algebra` package.
+must be written using the same set of algebra interfaces, but with additional
+parameters supplied.
```

Comment:
I wouldn’t mention that we use the same set of algebra interfaces. Only the fact that generating the documentation just consists in applying another interpreter to our endpoint descriptions, and that we can enrich those descriptions with additional documentation information.